### ISSUE TITLE
pay invoices between two accounts using the same LND node.

### ISSUE THREAD

**Issue by AndySchroder** [lightninglabs/lightning-terminal#494]
**Is your feature request related to a problem? Please describe.**
In the documentation (https://docs.lightning.engineering/lightning-network-tools/lightning-terminal/accounts) it says "At this point, it is not possible to pay invoices between two accounts using the same LND node."

**Describe the solution you'd like**
Allowing paying invoices between two accounts using the same LND node.

**Describe alternatives you've considered**
`allow_self_payment` may work, but I'm not sure why you haven't mentioned it in the documentation. Even with `allow_self_payment`, I think that https://github.com/lightningnetwork/lnd/issues/7419 needs to be fixed first.

**Comment by ellemouton** [comment]
agreed that https://github.com/lightningnetwork/lnd/issues/7419 will need to be fixed first 👍

**Comment by ZZiigguurraatt** [comment]
> agreed that [lightningnetwork/lnd#7419](https://github.com/lightningnetwork/lnd/issues/7419) will need to be fixed first 👍


Based on https://github.com/lightningnetwork/lnd/issues/7419#issuecomment-4731445983, I don't think we are going to fix https://github.com/lightningnetwork/lnd/issues/7419 and the solution to this issue probably needs to come primarily from within litd.


Also, I think we should fix https://github.com/lightninglabs/lightning-terminal/issues/538 at the same time that this issue is fixed because we may need to use HOLD invoices for litd to have a chance to detect a self payment and it would be best to have a solid architecture that can fix both problems together .

**Comment by mentioneddd** [comment]
got it thanks
